client/layout/masterbar/logged-in.jsx

```diff
@@ -298,12 +305,77 @@ class MasterbarLoggedIn extends Component {
 
 	renderReader( showLabel = true ) {
 		const { translate } = this.props;
+		const readerIcons = [
+			/* First */
+			<svg
+				className="gridicon gridicons-reader"
```

Comment:
Needed `gridicon gridicons-reader` to apply correct styles, so prolly need to track down CSS to update to be able to use custom SVG.